astroquery/alma/README.rst

```diff
@@ -0,0 +1,62 @@
+===================
+Accessing ALMA data
+===================
```

Comment:
This file should not have been created. `docs/alma/alma.rst` should have been edited instead.